corehq/apps/app_manager/views/releases.py

```diff
@@ -250,3 +257,85 @@ def update_build_comment(request, domain, app_id):
     build.build_comment = request.POST.get('comment')
     build.save()
     return json_response({'status': 'success'})
+
+
+def _get_app_diff_files(app):
```

Comment:
Thanks for the suggestion. We could do that, but I wonder if it would be better to keep app.json off of that page because it's sort of in a different category as the other resource files. All the resource files are "attachements" to the app document in couch, and I think they are all of the files that are sent to the phone when installing an app, whereas app.json is not. I wonder if it is misleading to include it in the same list. Thoughts?